.travis.yml

```diff
@@ -23,6 +23,8 @@ before_script:
   - cp config/secrets.yml.example config/secrets.yml
   - mysql -u root -e 'create database osem_test;'
   - RAILS_ENV=test bundle exec rake db:migrate --trace
+  - gem install haml_lint
 script:
   - 'bundle exec rubocop -Dc .rubocop.yml'
+  - 'bundle exec haml-lint -c .haml-lint.yml app/views'
```

Comment:
`-c .haml-lint.yml` shouldn't be needed as `.haml-lint.yml` is the default config file.